source/wp-content/plugins/wporg-developer-blog-snippets/index.php

```diff
@@ -0,0 +1,119 @@
+<?php
```

Comment:
Can you rename this file to `wporg-developer-blog-snippets.php` or similar? I know `index` is nicer, but we end up with people trying to access these directories, so it would be better to rename (or add a check: `defined( 'WPINC' ) || die();`)